strawberryfields/backends/gaussianbackend/backend.py

```diff
@@ -201,16 +201,19 @@ def measure_fock(self, modes, shots=1, select=None):
                           "Conditional state after Fock measurement has not been updated.")
 
         mu = self.circuit.mean
+        mean = self.circuit.smeanxp()
         cov = self.circuit.scovmatxp()
```

Comment:
Out of curiosity, what is the difference between `self.circuit.mean` and `self.circuit.smeanxp`?

Reply:
self.circuit.mean returns alpha, smeanxp return x and p.